pkg/releases/releases.go

```diff
@@ -294,6 +305,24 @@ func ParseReleases(data string) (*ReleasesInfo, error) {
 		}
 	}
 
+	swiftMatches := swiftReleaseRegex.FindStringSubmatch(lastRelease)
+
+	if len(swiftMatches) == 5 {
```

Comment:
There is a _test.go file for releases that you should add swift in to make sure this all works as expected